src/bsp/bm_mote_rs232/Drivers/STM32U5xx_HAL_Driver/Inc/stm32u5xx_ll_usart.h

```diff
@@ -0,0 +1,4488 @@
+/**

```

Comment:
Until we make `serial.c` agnostic to USART v.s. LPUART, we need to make sure this stays in the bsp.